### DISCUSSION TITLE
Offer a stream variant "audio"

### DISCUSSION THREAD

**Discussion by luwimi** [owncast/owncast#2973] [Ideas]
For some use cases it would be helpful to have streaming profiles that only provide the video input as an audio stream. High-quality modern codecs with high bandwidth such as m4a, mp3 as a proven fallback and small bandwidth solutions such as OPUS would be desirable.

A bonus would be if the statistics differentiated between audio and video.

**Comment by TomSelander** [comment]
+1 I came here to request the same thing. This would be a quick win as I think it's just a different flag passed to ffmpeg (`-an`)
